README.md

```diff
@@ -18,6 +18,11 @@
 
 ## Quick Start
 
+### Requirements
+
+* iOS 12 or higher (or tvOS 12 or higher)
+* Swift 5.7 or higher
+
```

Comment:
Good idea, I have added the badges, thank you! 

But I would suggest using them in addition to the Requirments section, not instead of. This might be different in the iOS community, but I would never expect important information to be present only in the Readme badges, I usually glance right over them.

Reply:
Yes, in iOS-related repositories this kind of information is usually present in badges section and most people find it sufficient, because it can also be derived from the `Package.swift` and `FingerprintPro.podspec` files, which are always the ultimate source of truth. However, I am still okay with keeping the `Requirements` section (we will just need to remember to update the relevant information in two different places).

Given the above, one thing that still needs to be changed is this statement: "Swift 5.7 or higher". In its current shape it implies that we ensure the support even for the beta Swift releases (such as Swift 5.9 at the time of writing this comment), which is not true. In fact, we only state that we support the latest stable Swift toolchain once it is thoroughly tested. Moreover, as the Swift toolchain versions are strongly coupled with [Xcode releases](https://xcodereleases.com/), it would make more sense for the `Requirements` section to specify the minimum supported Xcode version. For this reason, I would suggest to change the aforementioned statement to: "Xcode 14.1 or higher". This is as per the latest [App Store submission requirement](https://developer.apple.com/news/?id=jd9wcyov).

Reply:
That makes sense, thank you! I have changed the requirement to `Xcode 14.1 or higher`. We can also leave it out completely if you prefer. 

I renamed the first commit message, but there are multiple commits now, so you might want to squash and merge anyway and you can specify the exact message you want. I don't really need to have the task identifier in the branch name, renaming the branch now would close this PR and I would need to create another one. But we can do that if you prefer.

Reply:
That's okay, we can leave the branch name as is.